### ISSUE TITLE
packages from old releases are not available on official repositories

### ISSUE THREAD

**Issue by linuxmaniac** [sipwise/kamailio-deb-jenkins#9]
reprepro can't handle more than one version of the same package so as soon as we release a new version older debs get removed from the repository

There's a really nice project [aptly](https://www.aptly.info/) that can handle more than one version per repo.

We need to investigate how to integrate aptly to [jenkins-debian-glue](http://jenkins-debian-glue.org/) in order to be able to provide all official previous releases

**Comment by mika** [comment]
I agree that we should look into aptly, thanks for reporting this here :)

JFTR though: https://www.preining.info/blog/2018/10/aptly-reprepro/ :-/

**Comment by mika** [comment]
This issue won't be taken care of here, further work takes place at https://github.com/kamailio/kamailio-infra + https://github.com/kamailio/jenkins-jobs, closing accordingly.
